### ISSUE TITLE
Need to verify if "get-help -online" works fine on the Azure code coverage VM

### ISSUE THREAD

**Issue by daxian-dbw** [PowerShell/PowerShell#3387]
#3051 introduced the `if((-not ($skipTest)) -and $IsWindows)` block in `test/powershell/engine/Help/HelpSystem.OnlineHelp.Tests.ps1` because our default-browser-discovery code, which was used in `Get-Help -Online`, didn't work correctly on the Azure code-coverage VM.

After #3281, that code was replaced by calling into `ShellExecuteEx`. We need to verify if `Get-Help -Online` works fine now on the Azure code-coverage VM, and if it does then this `if` block should be removed completely.

**Comment by daxian-dbw** [comment]
It turns out this is a duplicate of #3079
